repository/plackbot/Plackbot/1.0-SNAPSHOT/PlackBot-1.0-SNAPSHOT.jar.lastUpdated

```diff
@@ -0,0 +1,4 @@
+#NOTE: This is an Aether internal implementation file, its format can be changed without prior notice.
+#Tue Apr 24 19:18:38 UTC 2018
+file\:///home/jgoulah/dev/PushBot/repository/.error=
+file\:///home/jgoulah/dev/PushBot/repository/.lastUpdated=1524597518554
```

Comment:
This had me a little concerned, but I pulled down the branch and was successfully able to compile so... I guess this is fine?

Reply:
I built the plack jar separately and imported it.  Not sure what these lines actually mean/do